### PR TITLE
Added voice over accessibility support

### PR DESCRIPTION
**Pull request by JamesJSY**
The previous one has some conflicts with the master branch and I cannot update it, even with -f flag. So I created a new pr.

### CODE REVIEW

File: Pod/Classes/UI/CardView.xib [hunk 1]
```diff
@@ -48,6 +50,7 @@
                                 <textField opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="YY" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="CgE-Em-qHJ" customClass="YearInputTextField" customModule="Caishen" customModuleProvider="target">
                                     <rect key="frame" x="272" y="0.0" width="37" height="596"/>
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                    <accessibility key="accessibilityConfiguration" label="Year for Expiration Date"/>
```
Comment (ppatriotis) [inline Pod/Classes/UI/CardView.xib]: @Haud shouldn't this be "Expiration Year"

File: Pod/Classes/UI/CardView.xib [hunk 2]
```diff
@@ -39,6 +40,7 @@
                                 <textField opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="right" contentVerticalAlignment="center" placeholder="MM" textAlignment="right" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="5cX-Z5-afy" customClass="MonthInputTextField" customModule="Caishen" customModuleProvider="target">
                                     <rect key="frame" x="229" y="0.0" width="37" height="596"/>
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                    <accessibility key="accessibilityConfiguration" label="Month for Expiration Date"/>
```
Comment (ppatriotis) [inline Pod/Classes/UI/CardView.xib]: @Haud shouldn't this be "Expiration Month"

File: Pod/Classes/UI/CardView.xib [hunk 3]
```diff
@@ -63,6 +66,7 @@
                                 <textField opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="CVC" textAlignment="right" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="HOM-PC-B5m" customClass="CVCInputTextField" customModule="Caishen" customModuleProvider="target">
                                     <rect key="frame" x="309" y="0.0" width="221" height="596"/>
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                    <accessibility key="accessibilityConfiguration" label="CVC for Card"/>
```
Comment (ppatriotis) [inline Pod/Classes/UI/CardView.xib]: @Haud shouldn't this be "Card Verification Code"?

**Comment by ppatriotis** [comment]
How would translations to other languages work?

File: Pod/Classes/UI/CardView.xib (hunk 1 above)
Comment (Haud) [inline Pod/Classes/UI/CardView.xib]: Yea, that would make more sense.

File: Pod/Classes/UI/CardView.xib (hunk 2 above)
Comment (Haud) [inline Pod/Classes/UI/CardView.xib]: Yes

File: Pod/Classes/UI/CardView.xib (hunk 3 above)
Comment (Haud) [inline Pod/Classes/UI/CardView.xib]: Yup

**Comment by JamesJSY** [comment]
Thanks for pointing out @ppatriotis. Fixed accessibility label text and also added localization support for English, Chinese and German. Huge thanks to @DannyVancura :100: 

Users can either update the property accessibilityLabel on each textfield to fit their needs for voice over accessibility support. Or make a new pr to add new localizable.strings file to support their language.

**Comment by DannyVancura** [comment]
To consider this full voice over accessibility support we should think about audible notifications that the provided card number or expiration date were invalid (i.e. make color changes audible).

**Comment by JamesJSY** [comment]
Added audible notifications that the provided card number was invalid and the provided expiration date had expired. @ppatriotis @Haud 

Enormous thanks to @DannyVancura

File: Caishen.podspec [hunk 4]
```diff
@@ -29,5 +29,5 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.source_files = 'Pod/Classes/**/*'
-  s.resources = 'Pod/Assets/*'
+  s.resources = ['Pod/Assets/*', 'Pod/Resources/*']
```
Comment (Haud) [inline Caishen.podspec]: Why did we need a separate folder for this / what's the difference?

File: Section 508.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# Section 508
```
Comment (Haud) [inline Section 508.md]: Why was this added? Does anything link to this?

**Comment by Haud** [comment]
Where did the translations come from? Are they from native speakers?

**Comment by DannyVancura** [comment]
Yes. German translations from my end, Chinese from @JamesJSY. Both native speakers.

File: Section 508.md (hunk 5 above)
Comment (DannyVancura) [inline Section 508.md]: This just describes the compliance to governmental standards w.r.t. accessibility for visually impaired users, which is added in this PR.

File: Caishen.podspec (hunk 4 above)
Comment (DannyVancura) [inline Caishen.podspec]: The difference is more or less the kind of content. `Assets` for image resources and the like, whereas `Resources` in this case are mostly .strings files for the localization.
